packages/node/src/lib/process-request.ts

```diff
@@ -160,7 +160,8 @@ export default function processRequest(
   } else if (mimeType) {
     postData = {
       mimeType,
-      text: reqBody ? reqBody.toString() : null,
+      // Do our best to record *some sort of body* even if it's not 100% accurate.
+      text: typeof reqBody === 'string' ? reqBody : JSON.stringify(reqBody),
```

Comment:
I doubt `stringify` will throw, but theoretically it could

Reply:
sure enough it can: https://stackoverflow.com/questions/8106618/json-stringify-throws-error

I'll get a handler in there.

Reply:
Updated this to account for circular refs. Give it another once over.